chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/ActivityMetadataManager.java

```diff
@@ -49,4 +49,6 @@
     List<ActivityMetadata> getAnnualReportActivityMetadata(Date startDate, Date endDate) throws IOException;
 
     List<ActivityMetadata> getChangeRequestActivityMetadata(Date startDate, Date endDate) throws IOException;
+
+    List<ActivityMetadata> getApiKeyManagementMetaData(Date startDate, Date endDate) throws IOException;
```

Comment:
Can you make this method name capitalization consistent with the others? (Lowercase 'd" in Metadata)